packages/@pollyjs/core/src/-private/container.js

```diff
@@ -1,7 +1,19 @@
 import { assert } from '@pollyjs/utils';
 
+export function nameOfFactory(Factory) {
```

Comment:
Should this be `idOfFactory` or `factoryId`?

Reply:
Done